app/views/users/edit.html.erb

```diff
@@ -0,0 +1,31 @@
+<div class="col-md-5 bound-back">
+  <div class="row text-center">
+    <h2 class="text-title"><%= t ".edit_text" %></h2>
+  </div>
+  <%= form_for @user do |f| %>

```

Comment:
Please make a reusable form for both of the edit and new page and than render it here.